  
  #  [dictionary_id, board_w, board_h, square_length_in_meters, marker_length_in_meters]
  board_params: [0, 6, 4, 0.0232, 0.0174]

